utils/mfasweep.py

```diff
@@ -0,0 +1,156 @@
+from datetime import datetime
```

Comment:
Some overall thoughts: 

1. Biggest concern: Each of these MFA sweep requests will not be through a FireProx API, thus leaking your IP address. Generally fine, but poor opsec and not great for users who don't read the source
2. I sort of want to re-architect this in the future, ideally to have a "postex" type flag, almost like the plugins used. Would hopefully work in two ways, 1 used in-line during a spray so that a given postex module is ran on each set of creds found (like you have it set up), or 2 provide a username/password in the userpass file and _only_ run the postex module. Definitely a future plan, but just an FYI of my intentions

Reply:
1. The implementation I looked into wasn't possible to flip through a fireprox endpoint due to the nature of how MFASweep is working on the backend :( thus this option, suppose could implement an opsec warning for users.
2. I agree a postex module or flag setup makes sense